### PR TITLE
Preliminar fixing to meet syntax guidelines. It remains documentation…

### LINKED ISSUE #40: Add a linter to tests to ensure passing WordPress PHP code guidelines
- [x] Add linter: [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer)
  - [x] Set rules: [WordPress-Conding-Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards)
  - [x] Pass linter
  - [x] Add linter to Travis

### PR DESCRIPTION
**Pull request by delapuente**
… and refactor. The admin page will be refactored in another bug.

Fix #40

### CODE REVIEW

**Comment by delapuente** [comment]
@darkwing this first commit is aimed to fix syntax guidelines. I'm going to add documentation and make some suggested refactors. It could be nice if you try the plugin more than reviewing the changes (for that we have the linter!). Once all the warnings disappear I will add the linter to Travis.

What do you think?

File: wp-offline-content/class-wp-offline-shell-recommender.php [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
+	 */
 	public static function init() {
 		if ( ! self::$instance ) {
 			self::$instance = new self();
 		}
 		return self::$instance;
 	}
 
+	/**
+	 * Decides if there is a minified version of the file.
```
Comment (delapuente) [inline wp-offline-content/class-wp-offline-shell-recommender.php]: @darkwing could you clarify me what is this function doing, please?

**Comment by delapuente** [comment]
@darkwing I finished here. I tried it and it seems to be working but I would thank an extra pair of eyes looking at it.

The linter is not passing for class-wp-offline-shell-admin.php. This will be addressed when fixing #38

**Comment by darkwing** [comment]
Excellent!  R+ from me, feel free to merge if you'd like!

**Comment by marco-c** [comment]
Nice, we should do the same for the other plugins!

**Pull request merged**